_includes/topbar.html

```diff
@@ -35,7 +35,8 @@
                             <li><a href="media/pdf/ESGF-Implementation-Plan-V1.0.pdf">Implementation Plan</a></li>
                             <li><a href="logo_requirements.html">Logo Requirements</a></li>
                             <li><a href="https://github.com/ESGF/esgf.github.io/wiki/ESGF-Index-and-IdP-nodes">Index &
-                                IdP Nodes</a></li>
```

Comment:
This change is removing the ending tags on the list item above the one you added.